docs: Update remote-notification-support.md (#451)
invertase/notifee: docs-react-native/react-native/docs/ios/remote-notification-support.md

Before, moving to the next step, run the app and check it builds successfully – make sure you have the correct target selected. 

## Edit the payload
Now everything is setup in your app, you can alter your notification payload in two ways:

> 1. When creating the message contents for sending, in your back-end

Reviewer: ```suggestion
1. Update the message payload, sent via your backend
```

> 1. When sending from your BE (either via REST api or firebase admin sdk)

Reviewer 2: Most people integrate via firebase, but really, on iOS at least it could be anything that delivers to the device eventually, so making it a little more generic

```suggestion
1. When creating the message contents for sending, in your back-end
```

> 2. In Notification Service Extension when you receive notification

Reviewer 2: I have difficulty troubleshooting sometimes because messages and notifications etc are used interchangeably when they actually are specific and different, so I'm just being pedantic here by being specific with "remote message" but it may help future troubleshooting

```suggestion
2. In a Notification Service Extension in your app when a device receives a remote message
```

> Make sure that you will set `mutable-content: 1` (mutableContent if you are using firebase admin sdk) when sending notification otherwise Notification Service Extension will NOT be triggered

> Make sure that you will set `content-available: 1` (contentAvailable if you are using firebase admin sdk) if you want to receive notification when your app is in foreground

> ### 1. When sending from your BE

Reviewer 2: Just expanding the acronym so it is more obvious / less jargon

```suggestion
### 1. When sending from your back-end
```

> ### 1. When creating the message contents for sending, in your back-end

Reviewer: ```suggestion
### 1. Update the message payload, sent via your backend
```
